src/content/docs/mobile-apps/android.mdx

```diff
@@ -14,14 +14,9 @@ redirects:
   - /docs/mobile-apps/new-relic-mobile-apps/android-app/introduction-new-relic-android-app
 ---
 
-import iconIosClock0 from 'images/icon-ios-clock_0.png'
+import mobileappsAndroidAppAPMDashboard from 'images/mobile-apps_screenshot-full_Android-app-APM-dashboard.png'
 
-The user interface for New Relic's Android app provides functionality similar to New Relic's standard user interface, with customized details for mobile users. Learn more from the [New Relic website](https://newrelic.com/mobility). 
-
-## Compatibility and requirements [#reqs]
-
-* Android 8.0 (Oreo) or higher
-* Screen size of 7 inches or less
+New Relic's Android app lets you access your data wherever you are. You can view curated dashboards from across New Relic capabilities, from APM to infrastructure to browser, and access any dashboards you've built. If you need to go deeper, you can query your data directly with NRQL or search your logs. And you can receive alerts directly on the app so you can start investigating immediately.
```

Comment:
Sat in on many calls between the VM team and legal. They (legal) really want us avoiding  `New Relic's` wherever possible. I'd replace all instances of `New Relic's Android app` w/ `The New Relic Android app`

I'll mark all places I find it with  `Here`

Reply:
Style guide reference: https://docs.newrelic.com/docs/style-guide/capitalization/product-capability-feature-usage/